noncompiled/view/ScalarDisplay.js

```diff
@@ -132,11 +147,11 @@ scalarDisplays.npp 	= new ScalarDisplay( { color: 0x00ff00, min: '0.', max: '1.'
 			return npp;
 		} 
 	} );
-scalarDisplays.alt 	= new ScalarDisplay( {color: 0x000000, min:'sealevel', max:'maxheight', 
+scalarDisplays.alt 	= new ScalarDisplay( { minColor: 0x000000, maxColor: 0xffffff, min:'0.', max:'30000.', 
 		getField: function (world, result) {
-			ScalarField.sub_scalar(world.displacement, world.SEALEVEL, result);
-			return result
-		} 
+			return (scalarDisplayVue.ocean)?(ScalarField.max_scalar(world.displacement, world.SEALEVEL)):
```

Comment:
glad to see you find the ScalarField namespace useful

Reply:
Also I tried this out. When you toggle oceans, it reflects the change correctly in the chart, but the map doesn't change appearance. I think this is because we would also need to change the values for `max` and `min`. There's a few things we could do to accomplish this and I'm open to discussion.

Reply:
If you look carefully, the map does change, but it's subtle. At least, I'm seeing it.

Reply:
Nope, nevermind. It does change the map but it's hard to tell because the max value for the Display is set so high. Now if oceans are toggled they appear a little greyer, which may or may not be intended behavior. Let me know.